tests/notebooks/test_moving_between_models.py

```diff
@@ -0,0 +1,25 @@
+from testbook import testbook
+from dffml import IntegrationCLITestCase
+
+
+class TestNotebook(IntegrationCLITestCase):
+    @testbook("examples/notebooks/Moving_between_models.ipynb")
+    def test_stdout(tb, self):
+        tb.execute_cell(range(0, 5))
+        tb.inject(
+            """
+            data_path = await cached_download(
+                "https://archive.ics.uci.edu/ml/machine-learning-databases/wine-quality/winequality-red.csv",
+                "tests/notebooks/data/wine_quality.csv",
```

Comment:
As you suggested previously to mock the call to `cached_download` to give it a different path and gitignore that directory we download the example data to. Inject lets you to add some code to a notebook and execute it as part of the notebook, so I ignored the cell execution of the actual `cached_download` call and instead injected and executed this mock call.